### PR TITLE
Upgraded to Angular 6

### PR DESCRIPTION
**Pull request by nfavaron**
Our Angular 6 app using `ng2-ui/auto-complete` runs without `rxjs-compat`.

Therefore we need `ng2-ui/auto-complete` to work on rxjs 6 and angular 6 as well :)

The update is pretty straight forward, let me know if you need more info.

### CODE REVIEW

File: app/app.service.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/catch';
+import { Observable } from 'rxjs';
```
Comment (almothafar) [inline app/app.service.ts]: Why you changed those? is everything still working and tested in demo app after this?

File: package.json [hunk 2]
```diff
@@ -18,20 +18,21 @@
     "tslint": "tslint"
   },
   "dependencies": {
+    "@angular/cli": "^6.0.3",
```
Comment (almothafar) [inline package.json]: This should be in devDependencies , not in dependencies

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
+### [1.0.3] 
+- Upgraded to Angular 6
```
Comment (almothafar) [inline CHANGELOG.md]: Angular 6 should be 1.2.0, there is ongoing 1.1.0 fixes https://github.com/ng2-ui/auto-complete/milestone/1

File: app/app.service.ts (hunk 1 above)
Comment (nfavaron) [reply app/app.service.ts]: With RxJS 6, we now need to import operators only in files that need it, here is a good example: https://github.com/ng2-ui/auto-complete/pull/351/files#diff-8aaf19bd0c967a0eaa473a82229fe2d8R4

File: package.json (hunk 2 above)
Comment (nfavaron) [reply package.json]: Good catch, dunno why it can in dependencies instead of devDependencies.

File: CHANGELOG.md (hunk 3 above)
Comment (nfavaron) [reply CHANGELOG.md]: Done.

**Comment by nfavaron** [comment]
Demo is working, just had to rename a material design component :)

**Comment by webcat12345** [comment]
Looking forward to see this feature!

**Pull request merged**